src/components/BackToQwantButton.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Button } from 'src/components/ui';
+
+export const BackToQwantButton = ({ ...props }) => {
+
+  return (
+    <Button
+      className="backToQwantButton"
+      icon="arrow-left"
+      variant="tertiary"
+      onClick={() => window.history.back()}
+      {...props}
+    >
+      Retour sur Qwant
```

Comment:
To translate, and according to mockups, the wording should be something like "Back to Qwant.com".